Objects/genericaliasobject.c

```diff
@@ -118,12 +126,17 @@ ga_repr_item(_PyUnicodeWriter *writer, PyObject *p)
 static PyObject *
 ga_repr(PyObject *self)
 {
-    gaobject *alias = (gaobject *)self;
+    gaobject *alias = (gaobject *) self;
```

Comment:
No need for this change. This reappears a couple of times below, the style I've seen throughout the CPython C code has been no space after the parenthesis in a cast.

Reply:
Gotcha; thanks! Fixed.